src/gc/pal/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
I moved the contents of "src/gc/pal" to "src/gc/unix" and moved the Windows-specific code to "src/gc/windows" in the last commit, is that what you had in mind?